casper/src/main/scala/coop/rchain/casper/merging/DeployChainIndex.scala

```diff
@@ -50,4 +61,19 @@ object DeployChainIndex {
       stateChanges
     )
   }
+
+  def random: Iterator[DeployChainIndex] =
```

Comment:
we can put this function in tests

Reply:
I encountered a problem with this when trying to create benchmark suite - you cannot easily reference test code from main project. So if defined in test, you cannot somewhere in benchmark code inside node project use it. Might be pushing benchmark suite in main project is not a good idea, but I'm not sure, as it would be nice to have the same binary for benchmarks and for node operation.